Lux/src/next_element.cpp

```diff
@@ -379,6 +379,57 @@ void next_element::add_function(  any_gen_fn fn       ) { functions.push_back( f
 
 void next_element::add_condition( any_condition_fn c  ) { conditions.push_back( c ); }
 
+// Audio function implementations - moved here to access full element_context definition
+
+float audio_additive_fn::operator() ( float& val, element_context& context) {
```

Comment:
It might be better here just to have a pure audio function and add values via a harness or adder instead of multiple slightly different audio functions.